doc/manual/manual.tex

```diff
@@ -1419,94 +1645,20 @@ \subsubsection{The Boussinesq approximation: Incompressibility}
 different, however, for models in which the density changes rather abruptly,
 for example due to phase changes at mantle discontinuities.
 
-In summary, models that use the approximation of incompressibility solve the
-following set of equations instead of \eqref{eq:stokes-1}--\eqref{eq:temperature}:
-\begin{align}
-  \label{eq:stokes-1-boussinesq}
-  -\nabla \cdot \left[2\eta \varepsilon(\mathbf u)
-                \right] + \nabla p &=
-  \rho \mathbf g
-  & \qquad
-  & \textrm{in $\Omega$},
-  \\
-  \label{eq:stokes-2-boussinesq}
-  \nabla \cdot \mathbf u &= 0
-  & \qquad
-  & \textrm{in $\Omega$},
-  \\
-  \label{eq:temperature-boussinesq}
-  \rho C_p \left(\frac{\partial T}{\partial t} + \mathbf u\cdot\nabla T\right)
-  - \nabla\cdot k\nabla T
-  &=
-  \rho H
-  +
-  2\eta
-  \varepsilon(\mathbf u)
-  :
-  \varepsilon(\mathbf u)
-  & \quad
-  & \textrm{in $\Omega$},
-\end{align}
-where the coefficients $\eta,\rho,\mathbf g,C_p$ may possible depend on the
-solution variables.
-
-\note{As we will see in Section~\ref{sec:extending}, it is easy to add new material
-models to \aspect. Each model can decide whether it wants to use the
-Boussinesq approximation or not. The description of the models in
-Section~\ref{parameters:Material_20model} also gives an answer which of the
-models already implemented uses the approximation or considers the material
-sufficiently compressible to go with the fully compressible continuity equation.}
-
-
-\subsubsection{Almost linear models}
+\paragraph{On almost linear models.}
```

Comment:
It seems like this paragraph has mostly become obsolete. Or do you think it's worth keeping?